### DISCUSSION TITLE
Where should we be writing feature requests?

### DISCUSSION THREAD

**Discussion by Manueljlin** [uxbox/uxbox#3030] [General]
There's three different places currently for the same thing:
- community.penpot.app
- GitHub discussions
- GitHub issues

**Comment by niwinz** [comment]
github discussions are deprecated,
if you have github account, github issues is the best place, but for make it more easy, the community is also allowed and we replicate ourselves it to github/taiga if proceed.

**Reply by niwinz** [reply]
On discussions you have a big banner saying it: 
![image](https://user-images.githubusercontent.com/843689/224312850-c5f86945-59cf-4454-8e2b-d1d35722cb72.png)

**Reply by Manueljlin** [reply]
my eyes kind of ignored the giant banner for some reason lol, sorry
